omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java

```diff
@@ -102,6 +108,43 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 		}
 	}
 	
+	@Override
+	public Model getCREATEModel(Representation rep) {
+		ModelImpl model = new ModelImpl()
+		        .property("encounter", new StringProperty().example("uuid"))
+		        .property("action", new StringProperty()
+		                ._enum(SwaggerSpecificationCreator.getEnumsAsList(Order.Action.class)))
+		        .property("accessionNumber", new StringProperty())
+		        .property("dateActivated", new DateProperty())
+		        .property("scheduledDate", new DateProperty())
+		        .property("patient", new StringProperty().example("uuid"))
+		        .property("concept", new StringProperty().example("uuid"))
+		        .property("careSetting", new StringProperty().example("uuid"))
+		        .property("dateStopped", new DateProperty())
+		        .property("autoExpireDate", new DateProperty())
+		        .property("orderer", new StringProperty().example("uuid"))
+		        .property("previousOrder", new StringProperty().example("uuid"))
+		        .property("urgency", new StringProperty()
+		                ._enum(SwaggerSpecificationCreator.getEnumsAsList(Order.Urgency.class)))
+		        .property("orderReason", new StringProperty().example("uuid"))
+		        .property("orderReasonNonCoded", new StringProperty())
+		        .property("instructions", new StringProperty())
+		        .property("commentToFulfiller", new StringProperty())
+		        
+		        .required("orderType").required("patient").required("concept");
+		if (rep instanceof FullRepresentation) {
+			model
+			        .property("encounter", new RefProperty("#/definitions/EncounterCreate"))
+			        .property("patient", new RefProperty("#/definitions/PatientCreate"))
+			        .property("concept", new RefProperty("#/definitions/ConceptCreate"))
+			        .property("orderer", new RefProperty("#/definitions/UserCreate"))
+			        .property("previousOrder", new RefProperty("#/definitions/OrderCreate"))
+			        .property("orderReason", new RefProperty("#/definitions/ConceptCreate"));
+		}
+		//FIXME missing prop: type
```

Comment:
Mention this in the next issue you create where you will be fixing the changes you removed formatting, etc not directly relevant to this issue